app/src/main/java/com/github/ybecker/epforuml/QuestionTextType.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.ybecker.epforuml
+
+enum class QuestionTextType(private val v: String) {
```

Comment:
I think enums have a name attribute so you could have just defined an enum with TEXT and LATEX without anything else.

Reply:
What I meant is that enums already have a Enum.name attribute so you could have for example TEXT.name which gives you the string "TEXT" or something similar so we wouldn't need to pass a text argument and a getter method.